code-block/README.md

```diff
@@ -29,7 +29,11 @@ In the Block Library, select a block to edit. Add a new field, set the _Field Ty
 Configure the options:
 
 - `enableTitle`: (optional) if set to `true`, users can provide a title (`content.title`).
-- `enableLanguage`: (optional) if set to `true`, users can provide a language (`content.language`).
+- `languages`: (optional) a JSON array of strings. The strings are the languages that the user can choose between.
+  ```json
+  ["JavaScript", "TypeScript", "Elm", "Vue", "Rust", "JSON"]
+  ```
+  The options will be sorted alphabetically. If this option is omitted, the language select input is hidden.
```

Comment:
Maybe we shouldn't sort. If they want the list sorted alphabetically, they could put a sorted list. But, there can be a case where they want to list languages in the order of most frequent usages. What do you think?

Reply:
Good point... okay I will remove the sorting

Reply:
I actually do. If we want to support that too, then I think we will end up using Autocomplete-like component: https://mui.com/material-ui/react-autocomplete/